### ISSUE TITLE
Icons broken due to missing 'spectrum-icons.svg'

### ISSUE THREAD

**Issue by undavide** [adobe/spectrum-css#60]
### Expected Behavior
Icons show up in the proper `html` when the `loadIcons()` function is called, passing as the parameter the path of e.g. the `'spectrum-css-icons.svg'` or `'spectrum-css-icons-medium.svg'` (as suggested in the doc)

### Actual Behavior
They don't. Actually, running Gulp in the current repo produces a `dist/docs` page where the icons are missing too. As a side note, the readme mentions `AdobeSpectrum.loadIcons()`, while the currently provided function is just `loadIcons()`

### Reproduce Scenario (including but not limited to)
Run Gulp and open `dist/docs` – the only icon that works (e.g. in the Action Button) is the one where the `svg` is passed literaly.

#### Spectrum-CSS version
2.7.2

#### Sample Code that illustrates the problem
```
<button class="spectrum-ActionButton">
    <svg class="spectrum-Icon spectrum-Icon--sizeS spectrum--medium" focusable="false" aria-hidden="true"
        aria-label="Edit">
        <use xlink:href="#spectrum-icon-18-Edit"></use>
    </svg>
    <span class="spectrum-ActionButton-label">Edit</span>
</button>
```
#### Workaround

Inspecting the way the online doc page works (as opposed to the repo's gulp generated doc), the file that is passed to `loadIcons()` is **`'spectrum-icons.svg'`**, and not `'spectrum-css-icons.svg'`. Problem is that such file is not found anywhere in the repository.

Dowloading it from the online page and using it locally, icons finally do display!

Thanks,

Davide

**Comment by lazd** [comment]
@undavide you're right, the "workflow" icons in `spectrum-icons.svg` (those not required to render the component, but rather inserted by users where required to fit their usage of the component) are not currently released in a way that we can make them available for local builds. The UI icons in `spectrum-css-icons.svg` (those required to render components) are included in the repository and available for local builds, as you noted. We're working with the icons team at Adobe to come up with a set of icons we can properly open source, and we'll switch Spectrum CSS over to use those icons once available.

Also, the README does have incorrect paths and function names for loading icons. We'll get that fixed up soon (#63).

**Comment by Harbs** [comment]
Resolving this would be really helpful to me.

I'm about half-way through completing a [Spectrum component set for Royale](https://github.com/unhurdle/spectrum-royale/issues/5).

It would be nice to be able to use the icons to display the results when I'm done.

**Comment by Harbs** [comment]
Some of these are needed for basic functionality. One of them is CycleButton.

I'm pretty sure I have this working, but I can't really see because I don't have the assets.
https://github.com/unhurdle/spectrum-royale/blob/master/Spectrum/src/com/unhurdle/spectrum/CycleButton.as#L29

**Comment by lazd** [comment]
Fixed by #266
